PMM-8874 Updated modules for PMM 2.23

percona-csalguero opened the pull request:
Updated mongodb_exporter, postgres_exporter and toolkit.
Not included yet because we didn't talk about creating a new release for these exporters.

- proxysql_exporter: release-1.1.2
- rds_exporter: release-0.7.2

I made the branches with the changes. Not need to test because they are based on the corresponding master braches but there were not official releases for proxysql_exporter rds_exporter even when we have changes.

percona-csalguero committed:
PMM-8874 Updated modules for PMM 2.23

Updated mongodb_exporter, postgres_exporter and toolkit.
Not included yet because we didn't talk about creating a new release for
these exporters.

- proxysql_exporter: release-1.1.2
- rds_exporter: release-0.7.2

I made the branches with the changes. Not need to test because they are
based on the corresponding master braches but there were not official
releases for proxysql_exporter rds_exporter even when we have changes.

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-2067-eef6a54
client docker - perconalab/pmm-client-fb:PR-2067-eef6a54
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2067-eef6a54.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2067-eef6a54&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2067-eef6a54.tar.gz

denisok commented on .gitmodules:
```diff
@@ -25,11 +25,11 @@
 	path = sources/postgres_exporter/src/github.com/percona/postgres_exporter
 	url = https://github.com/percona/postgres_exporter
-	branch = master
+	branch = release-0.4.8
```
https://github.com/percona/postgres_exporter/pull/51

this probably should be merged to master and master stay ?

percona-csalguero committed:
PMM-8874 Using postgres_exporter from master

I've merged PMM-8787 to be able to test directly using master branch so
we can test all tickets together. If there are any problem, we should
revert PMM-8787 merge for PMM 2.23

JNKPercona commented:
server docker - perconalab/pmm-server-fb:PR-2067-9d7204e
client docker - perconalab/pmm-client-fb:PR-2067-9d7204e
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2067-9d7204e.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2067-9d7204e&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2067-9d7204e.tar.gz
